index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <header class="app-header">
+        <a href="#" class="logo">Narson</a>
+        <nav class="ambony">
+            <a href="#" class="humburger">☰</a>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please add the correct style to your `Hamburger` button like [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=04rBLu4GidVN0Uk6-0) 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/228303299-536af687-14d6-47fe-a900-a2796fba2679.png)